src/app/forms/custom-sections/tyres/tyres.component.ts

```diff
@@ -41,24 +47,87 @@ export class TyresComponent implements OnInit, OnDestroy, OnChanges {
   get template() {
     switch (this.vehicleTechRecord.vehicleType) {
       case VehicleTypes.PSV:
-        return getTyresSection(VehicleTypes.PSV);
+        return tyresTemplatePsv;
       case VehicleTypes.HGV:
-        return getTyresSection(VehicleTypes.HGV);
+        return tyresTemplateHgv;
       case VehicleTypes.TRL:
-        return getTyresSection(VehicleTypes.TRL);
+        return tyresTemplateTrl;
     }
   }
 
+  get isPsv(): boolean {
+    return this.vehicleTechRecord.vehicleType === VehicleTypes.PSV;
+  }
+
+  get isHgvOrTrl(): boolean {
+    return this.vehicleTechRecord.vehicleType === VehicleTypes.HGV || this.vehicleTechRecord.vehicleType === VehicleTypes.TRL;
+  }
+
   get types(): typeof FormNodeEditTypes {
     return FormNodeEditTypes;
   }
 
+  get widths(): typeof FormNodeWidth {
+    return FormNodeWidth;
+  }
+
+  get speedCategorySymbol(): MultiOptions {
+    return speedCategorySymbolAsOptions();
+  }
+
+  get fitmentCode(): MultiOptions {
+    return fitmentCodeAsOptions();
+  }
+
   get axles(): CustomFormArray {
     return this.form.get(['axles']) as CustomFormArray;
   }
 
+  getAxleTyres(i: number): CustomFormGroup {
+    return this.axles.get([i, 'tyres']) as CustomFormGroup;
+  }
+
+  getTyresRefData(tyre: Tyres, axleNumber: number) {
```

Comment:
This kinda sucks, I'd really like to hear better ways around this if anyone has them